src/hidimstat/utils.py

```diff
@@ -22,19 +23,42 @@ def quantile_aggregation(pvals, gamma=0.5, gamma_min=0.05, adaptive=False):
         return _fixed_quantile_aggregation(pvals, gamma)
 
 
-def fdr_threshold(pvals, fdr=0.1, method="bhq", reshaping_function=None):
-    if method == "bhq":
-        return _bhq_threshold(pvals, fdr=fdr)
-    elif method == "bhy":
-        return _bhy_threshold(pvals, fdr=fdr, reshaping_function=reshaping_function)
-    elif method == "ebh":
-        return _ebh_threshold(pvals, fdr=fdr)
-    else:
-        raise ValueError("{} is not support FDR control method".format(method))
+def _fixed_quantile_aggregation(pvals, gamma=0.5):
+    """Quantile aggregation function based on Meinshausen et al (2008)
+
+    Parameters
+    ----------
+    pvals : 2D ndarray (n_sampling_with_repetition, n_test)
+        p-value (adjusted)
```

Comment:
What does "adjusted" mean here ?

Reply:
I don't know. I didn't touch this function for the moment.
I think it's corrected.

Reply:
I'd remove it, because AFAICT it's not standard wording.

Reply:
The term `adjusted` is coming from the paper of MEINSHAUSEN et al. [1]

[1]  Meinshausen, N. 2008. “Hierarchical Testing of Variable Importance.” Biometrika 95 (2): 265–78. https://doi.org/10.1093/biomet/asn007.

Reply:
see PR#132

Reply:
From what I understand, adjusted refers to the equation (2.1) in “P -Values for High-Dimensional Regression.”. 
In the two-step procedure introduced by the authors:
 1. split the data in 2 a screening is run on the first set (aiming to reduce the dimension from d>n to d'<n). 
 2. Then, p-values are estimated using the second set. These p-values are then **adjusted** by a factor proportional to the cardinal of the ensemble obtained during the screening stage. This correction is important to control the FWER. 

 - Unless we implement precisely their method, I suggest that we remove the "(adjusted)" which is misleading. 
 - Should we implement their method?

Reply:
see PR #132